Monika After Story/game/zz_hotkeys.rpy

```diff
@@ -216,6 +236,8 @@ init python:
         config.keymap["dec_musicvol"] = [
             "K_MINUS","shift_K_UNDERSCORE","K_KP_MINUS"
         ]
+        config.keymap["derandom_topic"] = ["x","X"]
+        config.keymap["bookmark_topic"] = ["b","B"]
```

Comment:
have alpha check if hitting these keys during games that use keys (piano / hangman) breaks anything